src/main/java/com/quizapp/enterprise/controllers/QuizController.java

```diff
@@ -23,7 +23,7 @@
     @RequestMapping("/quizzes")
     public String userQuizes(Model model, Principal principal){
 
-        model.addAttribute("username",principal.getName());
+        model.addAttribute("username", principal.getName());
```

Comment:
added space after argument within parameters for best practices and readability.
This is recommended by the "Spacing & line breaks" section of the java best practices summary referenced below.

source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3